OrbitQt/orbitcodeeditor.cpp

```diff
@@ -265,13 +274,23 @@ void OrbitCodeEditor::updateLineNumberArea(const QRect& rect, int dy) {
   if (rect.contains(viewport()->rect())) updateLineNumberAreaWidth(0);
 }
 
+//-----------------------------------------------------------------------------
+void OrbitCodeEditor::UpdateHeatMapArea(const QRect& rect, int dy) {
+  if (dy)
```

Comment:
Nit: Add {} for "if" and for "else".

Reply:
Thanks. Also added it to the function above.